docker/package/packages.py

```diff
@@ -486,11 +507,13 @@ def mk_rollup_package(name, type, proto):
     types = ["tx"]
     packages = ["node", "client"]
     return [
-        mk_rollup_package(name, type, proto)
+        {f"tezos-{type}-rollup-{name}-{proto}": mk_rollup_package(name, type, proto)}
         for name in packages
         for type in types
         for proto in active_protocols
     ]
 
 
 packages.extend(mk_rollup_packages())
+
+packages = dict(ChainMap(*packages))
```

Comment:
This looks a bit odd to me, what's the purpose of using a `ChainMap` if it's immediately given to `dict` again?

Reply:
analogue of `foldr (//) empty`

Reply:
:thinking: why not use a single dict and add single elements to it, or join other dicts with it, instead? :thinking:

Reply:
I just wanted to change as much existing code as possible

Reply:
@krendelhoff2 I don't see how that would be a bigger difference tho :thinking: 

I mean, you have changed the code to make `packages` a list, then define this list:
```python
packages = [
    {
        "tezos-somethingA": TezosBinaryPackage(<definiton_here>)
    },
    {
        "tezos-somethingB": TezosBinaryPackage(<definiton_here>)
    },
    {
        "tezos-somethingC": TezosBinaryPackage(<definiton_here>)
    },
    {
        "tezos-somethingD": TezosBinaryPackage(<definiton_here>)
    },
]
```
or add an element to it:
```python
packages.append(
    {
        "tezos-somethingE": TezosBinaryPackage(<definiton_here>)
    }
)
```

What I'm suggesting is to have `packages` be a dictionary to begin with, and the two examples above become:
```python
packages = {
  "tezos-somethingA": TezosBinaryPackage(<definiton_here>),
  "tezos-somethingB": TezosBinaryPackage(<definiton_here>),
  "tezos-somethingC": TezosBinaryPackage(<definiton_here>),
  "tezos-somethingD": TezosBinaryPackage(<definiton_here>),
}
```
and
```python
packages["tezos-somethingE"] = TezosBinaryPackage(<definiton_here>)
```
respectively, which seems even more compact to me (and doesn't need `ChainMap` at all).

Don't you think so? If you do, since this has been merge already, please open another PR